docs/extensibility/form-section.md

```diff
@@ -311,30 +284,26 @@ ResourceRef widgets render two dropdowns to select the associated resources' nam
 
 #### Example
 
-```json
-[
-  {
-    "path": "spec.my-data",
-    "widget": "ResourceRef",
-    "resource": {
-      "kind": "Secret",
-      "version": "v1"
-    },
-    "provideVar": "secret",
-    "children": [{ "path": "key", "enum": "$keys($secret.data)" }]
-  },
-  {
-    "path": "spec.my-gateways",
-    "widget": "ResourceRef",
-    "resource": {
-      "kind": "Gateway",
-      "group": "networking.istio.io",
-      "version": "v1alpha3"
-    },
-    "toInternal": "( $values := $split($, '/'); { 'namespace': $values[0], 'name': $values[1] } )",
-    "toExternal": "namespace & '/' & name"
-  }
-]
+```yaml
+- path: spec.my-data
+  widget: ResourceRef
+  resource:
+    kind: Secret
+    version: v1
+  provideVar: secret
+  children:
+    - path: key
+      enum: '$keys($secret.data)'
+- path: spec.my-gateways
+  widget: ResourceRef
+  resource:
+    kind: Gateway
+    group: networking.istio.io
+    version: v1alpha3
+  toInternal:
+    "( $values := $split($, '/'); { 'namespace': $values[0], 'name': $values[1]
+    } )"
```

Comment:
I think it would be cleaner to use the `|` notation here
```suggestion
  toInternal: |
    (
      $values := $split($, '/');
      { 'namespace': $values[0], 'name': $values[1] }
    )
```